<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultySelector</class>
 <widget class="QDialog" name="DifficultySelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>145</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Select Difficulty</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="EasyButton">
      <property name="text">
       <string>EASY</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="MediumButton">
      <property name="text">
       <string>MEDIUM</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="HardButton">
      <property name="text">
       <string>HARD</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
